packages/web/src/index.tsx

```diff
@@ -18,7 +20,12 @@ const tagManagerArgs = {
 
 TagManager.initialize(tagManagerArgs);
 
-ReactDOM.render(<App />, document.getElementById("root"));
+ReactDOM.render(
+  <Suspense fallback={null}>
```

Comment:
Was this addition only needed when we were loading the translations from separate files?

Reply:
Thanks for catching that. It isn't needed now that the translations are hardcoded.